development.md

```diff
@@ -1,8 +1,29 @@
 # Development Guide
 
-## Packaging
+Developers can use hatch to simplify set up of a pyosMeta development
+environment and executing development tasks.
+
+## Setting up a development environment
+
+### Install hatch
+
+1. Follow the [hatch installation instructions](https://hatch.pypa.io/latest/install/#installation).
+2. Open a new terminal and run `hatch --version` to verify that `hatch` is available.
+
+### Open a local dev environment using hatch
+
+In a terminal, enter:
+
+`hatch shell`
+
+You will now have an activated virtual environment.
 
-pyosMeta uses hatch and hatchling as it's build back end.
+[Hatch Environments](https://hatch.pypa.io/latest/environment/#entering-environments) explains
```

Comment:
oh i also created a tutorial for [hatch here ](https://hatch.pypa.io/dev/tutorials/environment/basic-usage/) on using environments. should we link to that? i tried to be comprehensive and i can add to it as needed if we want. ofek just merged it the other week ✨